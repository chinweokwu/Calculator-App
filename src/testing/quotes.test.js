import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/quotes';

describe('Snapshot test', () => {
  test('Snapshot for Quotes component', () => {
    const renderedComponent = renderer.create(<Quote />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
