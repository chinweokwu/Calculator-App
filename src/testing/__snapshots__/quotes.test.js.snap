// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Snapshot for Quotes component 1`] = `
<div
  className="quote"
>
  <div
    className="login-box"
  >
    <h3>
      
    </h3>
    <div
      className="author"
    >
      
    </div>
    <button
      onClick={[Function]}
      type="button"
    >
      <span />
      <span />
      <span />
      <span />
      Change Quotes
    </button>
  </div>
</div>
`;
